src/test_workflow/compare_benchmark/compare_test_runner.py

```diff
@@ -0,0 +1,62 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+import os
+import subprocess
+
+from test_workflow.compare_benchmark.compare_args import CompareArgs
+
+
+class CompareTestRunner:
+    def __init__(self) -> None:
+        self.container_name = "compare-container"
+
+    def run_comparison(self, compare_args: CompareArgs) -> None:
+        """
+        Perform the comparison between two test results using the OSB tool.
+        """
+
+        # construct the command to compare two test executions
+        command = (
+            f"docker run --name {self.container_name} "
+            "-v ~/.benchmark/benchmark.ini:/opensearch-benchmark/.benchmark/benchmark.ini "
+            f"opensearchproject/opensearch-benchmark:1.6.0 "
+            f"compare --baseline={compare_args.id1} --contender={compare_args.id2} "
+        )
```

Comment:
Just my 2cents:

We should assume user is running `opensearch-benchmark` command in an env with such command already available. We should not help user setup docker, especially if someone is using system such as windows that does not always support linux docker by default. Also, you are assuming user must have docker installed by default. And you probably need to take case of the cleanup issue if not using `--rm` in the run commands.

I would suggest just invote `opensearch-benchmark` call directly, and let user take care of their own env setup. You could, however, mention in your readme file and refer to `opensearch-benchmark` repo on how to set it up.